package.json

```diff
@@ -78,7 +78,7 @@
     "test": "node scripts/generate-pem && npm run test:tap && npm run test:node-fetch && npm run test:fetch && npm run test:cookies && npm run test:wpt && npm run test:websocket && npm run test:jest && npm run test:typescript",
     "test:cookies": "borp --coverage -p \"test/cookie/*.js\"",
     "test:node-fetch": "mocha --exit test/node-fetch",
-    "test:fetch": "npm run build:node && tap --expose-gc test/fetch/*.js && borp --coverage -p \"test/webidl/*.js\"",
+    "test:fetch": "npm run build:node && node --expose-gc --test test/fetch/*.js && borp --coverage -p \"test/webidl/*.js\"",
```

Comment:
Tests are failing on Windows due to this line. Node versions before `v21` do not support the glob pattern and Windows does not resolve the pattern whereas Linux does.

So, I tried a few things:

1. 
    ```
    borp --coverage -p "test/fetch/*.js"
    ```
    But now we aren't able to pass the `--expose-gc` flag, so `fetch-leak.js` which depends on gc fails.
    Also, strangely `issue-2171.js` also fails when run through borp, but not when run individually through `node --test`

2. 
    ```
    node --expose-gc node_modules/borp/borp.js --coverage -p "test/fetch/*.js"
    ```
    This does work, but it is worse in other ways (depends on a file name in installed dep)

### Options

We have a couple of options to go for:

1. Add `--expose-gc` flag directly to borp, so we are able to do
    ```
    borp --expose-gc --coverage -p "test/fetch/*.js"
    ```
    I am working with @mcollina [here](https://github.com/mcollina/borp/pull/4) to see if that can be done

2. Remove dependency from `--expose-gc` flag
    We can update `fetch-leak.js` to dynamically enable gc
    ```
    import { setFlagsFromString } from 'node:v8';
    import { runInNewContext } from 'node:vm';

    setFlagsFromString('--expose-gc');
    const gc = runInNewContext('gc');
    gc() // instead of calling global.gc() directly
    ```
    
Happy to hear anyone's thoughts on this!

Reply:
I'd prefer option 1, does `borp` has been updated already?

Reply:
@metcoder95 Yes just landed in `borp`, will update the code here to option 1